spring-cloud-task-samples/partitioned-batch-job/src/main/java/io/spring/JobConfiguration.java

```diff
@@ -72,7 +70,7 @@
 	private ConfigurableApplicationContext context;
 
 	@Autowired
-	private DelegatingResourceLoader delegatingResourceLoader;
+	private ResourceLoader resourceLoader;
```

Comment:
I'm surprised this would work, considering there should be more than 1 autowire candidate.